payload/platformsh/.platform/services.yaml

```diff
@@ -1,5 +1,5 @@
 mysqldb:
-    type: 'mysql:10.0'
+    type: 'mysql:10.2'
```

Comment:
maybe go with 10.1, we know there are some issues with 10.2+ and 5.7+ at least with legacy, so if people use legacy bridge it will break or be very slow in some cases.